src/lib/components/breadcrumbs.svelte

```diff
@@ -96,92 +99,89 @@
+                projectLinks.push({
+                    name: 'All projects',
+                    href: `${base}/organization-${selectedOrg.$id}`
+                });
+            }
+
+            return {
+                top: { title: 'Switch project', items: projectLinks },
+                bottom: { items: [createProjectItem] }
+            };
+        }
+
+        return {
+            top: { items: [createProjectItem] },
+            bottom: { items: [createProjectItem] }
+        };
+    }
+
+    function createOrganizationBottomSheet() {
+        return !selectedOrg
+            ? switchOrganization
+            : ({
+                  top: {
                       items: [
                           {
-                              name: 'Create project',
-                              trailingIcon: IconPlus,
-                              href: `${base}/organization-${selectedOrg?.$id}?create-project`
+                              name: 'Organization overview',
+                              href: `${base}/organization-${selectedOrg?.$id}`
                           }
                       ]
                   },
-        bottom:
-            selectedOrg?.projects.length > 1
-                ? {
-                      items: [
-                          {
-                              name: 'Create project',
-                              trailingIcon: IconPlus,
-                              href: `${base}/organization-${selectedOrg?.$id}?create-project`
-                          }
-                      ]
-                  }
-                : undefined
-    } satisfies SheetMenu;
+                  bottom:
+                      organizations.length > 1
+                          ? {
+                                items: [
+                                    {
+                                        name: 'Switch organization',
+                                        trailingIcon: IconChevronRight,
+                                        subMenu: switchOrganization
+                                    }
+                                ]
+                            }
+                          : {
+                                items: [
+                                    {
+                                        name: 'Create organization',
+                                        leadingIcon: IconPlus,
+                                        onClick: createOrg
+                                    }
+                                ]
+                            }
+              } satisfies SheetMenu);
+    }
 
     function onResize() {
         if ((organisationBottomSheetOpen || projectsBottomSheetOpen) && !$isSmallViewport) {
             organisationBottomSheetOpen = false;
             projectsBottomSheetOpen = false;
         }
     }
+
+    $: projectsBottomSheet = createProjectsBottomSheet();
+
+    $: organizationsBottomSheet = createOrganizationBottomSheet();
```

Comment:
this is not reactive. we need to pass the props we want it to be reactive with. otherwise its not difference to a `const`